week-9/Homework/mandatory/2-exercises/4-music-player.js

```diff
@@ -17,37 +17,76 @@ Optional 2: Can you implement the shuffle functionality for your music player?
 This means the order the songs are played in will be random, but each song will only play once.
 
  */
-
-
+let i;
 class MusicPlayer {
-    // Add your code here
-
+  constructor() {
+    this.songList = [];
+  }
+  add(song, singer) {
+    this.album = {};
+    this.album.song = song;
+    this.album.singer = singer;
+    this.songList.push(this.album);
+  }
+  play() {
+    if (this.songList.length == 0) {
```

Comment:
you can modify this condition to cover more states 
like 
if the song doesn't exist  (song index is not in the list or there is no list) then send a message else play